src/content/recipes/@mui/material.md

```diff
@@ -185,25 +185,17 @@ This is because Storybook only adds props to the controls table that are explici
 
 module.exports = {
   stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: [
-    '@storybook/addon-links',
-    '@storybook/addon-essentials',
-    '@storybook/addon-interactions',
-    '@storybook/preset-create-react-app',
-  ],
-  framework: '@storybook/react',
-  core: {
-    builder: '@storybook/builder-webpack5',
-  },
+  addons: ['@storybook/addon-essentials', '@storybook/addon-styling'],
+  framework: '@storybook/your-framework',
   typescript: {
-    check: false,
-    checkOptions: {},
     reactDocgen: 'react-docgen-typescript',
     reactDocgenTypescriptOptions: {
-      // speeds up storybook build time
-      allowSyntheticDefaultImports: false,
-      // speeds up storybook build time
-      esModuleInterop: false,
+      compilerOptions: {
+        // speeds up storybook build time
```

Comment:
@Integrayshaun just one small nit about this. Can you capitalize the "speeds" and "makes" and remove the extra "speed" as a matter of repeating ourselves?